stable/kubernetes-dashboard/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.networkPolicy -}}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "kubernetes-dashboard.fullname" . }}
+  labels:
+    app: {{ template "kubernetes-dashboard.name" . }}
+    chart: {{ template "kubernetes-dashboard.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    kubernetes.io/cluster-service: "true"
```

Comment:
Not sure it needs cluster-service here, or what effect that will have.

Reply:
Removed it. It only makes sense on Services.